ecl/hql/hqlexpr.cpp

```diff
@@ -9477,6 +9846,25 @@ CHqlMultiParentScope::CHqlMultiParentScope(IIdAtom * _name, ...)
     va_end(args);
 }
 
+IHqlExpression * CHqlMultiParentScope::lookupNearestSymbol(IIdAtom * searchName, unsigned lookupFlags, HqlLookupContext & ctx, NearestSymbol & nearest)
+{
+    unsigned penalty = 0;
+    NearestSymbol tempNearest(searchName);
+    CHqlScope::lookupNearestSymbol(searchName, lookupFlags, ctx, tempNearest);
+    nearest.compare(tempNearest, penalty);
```

Comment:
I'm unclear about this version of compare which takes penalty as a reference.  Please can you clarify the idea behind it.